Doc/using/mac.rst

```diff
@@ -2,190 +2,433 @@
+
+.. image:: mac_installer_06_summary.png
+
+If prompted to, double-click on the :command:`Install Certificates.command`
+icon or file in the |applications_folder_version| window to complete the
+installation.
+
+.. image:: mac_installer_07_applications.png
+
+This will open a temporary :program:`Terminal` shell window that
+will use the new Python to download and install SSL root certificates
+for its use.
+
+.. image:: mac_installer_08_install_certificates.png
+
+If ``Successfully installed certifi`` and ``update complete`` appears
+in the terminal window, the installation is complete.
+Close this terminal window and the installer window.
+
+A default install will include:
 
 * A |python_version_literal| folder in your :file:`Applications` folder. In here
-  you find IDLE, the development environment that is a standard part of official
+  you find :program:`IDLE`, the development environment that is a standard part of official
   Python distributions; and :program:`Python Launcher`, which handles double-clicking Python
-  scripts from the Finder.
+  scripts from the macOS `Finder <https://support.apple.com/en-us/HT201732>`_.
 
 * A framework :file:`/Library/Frameworks/Python.framework`, which includes the
   Python executable and libraries. The installer adds this location to your shell
-  path. To uninstall Python, you can remove these three things. A
-  symlink to the Python executable is placed in :file:`/usr/local/bin/`.
+  path. To uninstall Python, you can remove these three things.
+  Symlinks to the Python executable are placed in :file:`/usr/local/bin/`.
 
 .. note::
 
-   On macOS 10.8-12.3, the Apple-provided build of Python is installed in
-   :file:`/System/Library/Frameworks/Python.framework` and :file:`/usr/bin/python`,
-   respectively. You should never modify or delete these, as they are
-   Apple-controlled and are used by Apple- or third-party software.  Remember that
-   if you choose to install a newer Python version from python.org, you will have
-   two different but functional Python installations on your computer, so it will
-   be important that your paths and usages are consistent with what you want to do.
-
-IDLE includes a Help menu that allows you to access Python documentation. If you
-are completely new to Python you should start reading the tutorial introduction
-in that document.
-
-If you are familiar with Python on other Unix platforms you should read the
-section on running Python scripts from the Unix shell.
-
+   Recent versions of macOS include a :command:`python3` command in :file:`/usr/bin/python3`
+   that links to a usually older and incomplete version of Python provided by and for use by
+   the Apple development tools, :program:`Xcode` or the :program:`Command Line Tools for Xcode`.
+   You should never modify or attempt to delete this installation, as it is
+   Apple-controlled and is used by Apple- or third-party software.  If
```

Comment:
Nit: the "Apple-" is a bit confusing; my best suggestion is `s/Apple- or/Apple-provided or/` but I'm open to others.

Reply:
Agreed